### PR TITLE
Spelling corrections

### PR DESCRIPTION
**Pull request by e-dub**
Saw a couple minor spelling errors in error messages I was getting. Corrected these and others of the same type.

### CODE REVIEW

File: applications/ContactStructuralMechanicsApplication/tests/NightlyTests.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 import KratosMultiphysics.KratosUnittest as KratosUnittest
 import Kratos_Execute_Contact_Structural_Test as Execute_Test
 
-# This utiltiy will control the execution scope in case we need to acces files or we depend
```
Comment (loumalouomega) [inline applications/ContactStructuralMechanicsApplication/tests/NightlyTests.py:10]: We can correct the "utiltiy" word if we correct access...

**Review by loumalouomega — CHANGES_REQUESTED** [review]
Change "utiltiy" too

File: applications/ContactStructuralMechanicsApplication/tests/NightlyTests.py (hunk 1 above)
Comment (e-dub) [reply applications/ContactStructuralMechanicsApplication/tests/NightlyTests.py:10]: Good point. Done.

**Review by loumalouomega — APPROVED** [review]

**Comment by philbucher** [comment]
can this branch be deleted?

**Comment by e-dub** [comment]
Si!

**Pull request merged**